synapse/cortex.py

```diff
@@ -5345,12 +5361,13 @@ async def addNodes(self, nodedefs, view=None):
         The "props" or "tags" keys may be omitted.
 
         '''
-        async with await self.snap(view=view) as snap:
-            snap.strict = False
-            async for node in snap.addNodes(nodedefs):
-                yield node
+        if view is None:
```

Comment:
this api seems to still have use in the benchmark tool

Reply:
Yeah seems like this should just be using syn.nodes feed data though